### ISSUE TITLE
Deploy SEO Model Updates

### ISSUE THREAD

**Issue by benwbrum** [FreeUKGen/MyopicVicar#1902]
The SEO landing pages require a script to be run occasionally to update them.  Once the multi-modal code is deployed on production, I will write a cron job that runs this rake task.

(I'll also test our privacy settings against `test2` with `wget` while I'm at it.)

**Comment by richpomfret** [comment]
@benwbrum has this already been completed?

**Comment by Captainkirkdawson** [comment]
@benwbrum SEO and nofollow is in production

**Comment by benwbrum** [comment]
The production run of the SEO landing page creation task raised two issues:
1) After an apparently successful completion, nothing appeared under the `/open` landing page URL.  I need to research and diagnose this.
2) The task ran (albeit in `nice` mode) for about six hours.  We need to strategize on the best time/location for a long-running task to run, with input form @lemon-ukgen

**Comment by richpomfret** [comment]
@lemon-ukgen it looks like Ben may require your help here (unless you've already done so). Can you take a look when you can?

**Comment by lemon-ukgen** [comment]
Each of the UKGEN projects is allocated a week in every four for their database rebuilds, since that's how the "one" projects operated. I expect FreeREG2 hasn't needed this, but it's still scheduled. There's a [calendar which tracks which project's week it is](https://www.freebmd.org.uk/update.html).

During that week you're free to do computationally expensive stuff, customarily on the master server. We can disable searches on that server during your work.

Is that schedule sufficient?

If you need it more frequently please can your characterise the load? Is it a single thread? Does it need lots of memory? If you'd like me to measure it please let me know how to run it safely.

**Comment by richpomfret** [comment]
@benwbrum please see Lemon's comments above in relation to getting this story updated and completed.

**Comment by benwbrum** [comment]
That schedule sounds fine.  Once the PR is merged and deployed, I'll set up the cron jobs.

**Comment by richpomfret** [comment]
Closing this as now done (cron job aside - which is in the other story).
